op-node/rollup/types.go

```diff
@@ -65,6 +66,16 @@ type AltDAConfig struct {
 	DAResolveWindow uint64 `json:"da_resolve_window"`
 }
 
+type ClusterConfig struct {
+	// DependencySet represents the chains that are in the cluster.
+	DependencySet map[uint64][]*big.Int `json:"dependency_set"`
+	// SystemConfig represents the L1 SystemConfig contract addresses for each chain in the cluster.
+	SystemConfig map[*big.Int]common.Address `json:"system_config"`
```

Comment:
Note to self: needs a fix, use `eth.ChainID` type which is a flat value. The `*big.Int` is not safe as map key, since two different instances of the same chain ID number can have different pointers.